Index: megafuse-source/Makefile
===================================================================
--- megafuse-source.orig/Makefile
+++ megafuse-source/Makefile
@@ -49,7 +49,9 @@ clean:
 	rm -f $(OBJ) $(OUT)
 
 install: $(OUT)
-	mkdir -p /usr/share/doc/MegaFuse
-	cp FAQ.txt LICENSE.txt README.md megafuse.conf /usr/share/doc/MegaFuse/
-	cp megafuse.service megafuse@.service /lib/systemd/system/
-	cp $(OUT) /usr/bin/
+	mkdir -p $(PREFIX)/usr/share/doc/MegaFuse
+	cp FAQ.txt LICENSE.txt README.md megafuse.conf $(PREFIX)/usr/share/doc/MegaFuse/
+	mkdir -p $(PREFIX)/lib/systemd/system/
+	cp megafuse.service megafuse@.service $(PREFIX)/lib/systemd/system/
+	mkdir -p $(PREFIX)/usr/bin
+	cp $(OUT) $(PREFIX)/usr/bin/
